WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1572,7 +1572,17 @@ - (BOOL)hasFailedMedia
 
 - (BOOL)isMediaUploading
 {
-    return self.mediaInProgress.count > 0;
+    if (self.mediaInProgress.count == 0){
+        return NO;
+    }
+    
+    __block BOOL mediaUploading = NO;
```

Comment:
Since `enumerateKeysAndObjectsUsingBlock` requires a block, a `__block` var and since we don't really use the `key` parameter, why not just go for something simpler to read:

```objective-c
BOOL mediaUploading = NO;

for (NSProgress* progress in self.mediaProgress) {
    if (progress.totalUnitCount != 0){
        mediaUploading = YES;
        break; // also added this...
    }
}

return mediaUploading;
```

That way we have very short lines, no need to have a block or a `__block` variable.

I also added a `break;` directive.